docs/user_guide/config_options.md

```diff
@@ -1,44 +1,132 @@
+
+    `trunk`
+    : Boolean value set to `True` if this branch is the trunk for CABLE. Else set to `False`.
+
+    `share_branch`
+    : Boolean value set to `True` if this branch is under `branches/Share` in the CABLE SVN repository. Else set to `False`.
+
+    `patch`
+    : Additional branch specific namelist settings for `cable.nml`. Settings specified in `patch` get "patched" to the base namelist settings used for both branches. Any namelist settings specified here will overwrite settings defined in the default namelist file and or any settings defined in the science configurations.
+    : The `patch` key must be a dictionary like data structure that is compliant with the [`f90nml`][f90nml-github] python package.
+    : This key is **optional** and can be omitted from the config file. By default `patch` is empty and does not modify the namelist file.
+
+    Example:
+    ```yaml
+    realisations: {
+      0: { # head of the trunk
+        name: "trunk",
+        revision: -1,
+        trunk: True,
+        share_branch: False,
+      },
+      1: { # some development branch
+        name: "test-branch",
+        revision: -1,
+        trunk: False,
+        share_branch: False,
+        patch: {
+          cable: {
+            cable_user: {
+              FWSOIL_SWITCH: "Lai and Ktaul 2000"
+            }
+          }
+        }
+      }
+    }
+    ```
 
 `experiment`
 
-: Type of experiment to run. To choose from:
+: Type of experiment to run. Experiments are defined in the **NRI Land testing** workspace on [modelevaluation.org][meorg]. This key specifies the met forcing files used in the test suite. To choose from:
+
+    - [`forty-two-site-test`][forty-two-me]: to run simulations using 42 FLUXNET sites
+    - [`five-site-test`][five-me]: to run simulations using 5 FLUXNET sites
+    - `AU-Tum`: to run simulations at the Tumbarumba (AU) site
+    - `AU-How`: to run simulations at the Howard Spring (AU) site
+    - `FI-Hyy`: to run simulations at the Hyytiala (FI) site
+    - `US-Var`: to run simulations at the Vaira Ranch-Ione (US) site
+    - `US-Whs`: to run simulations at the Walnut Gulch Lucky Hills Shrub (US) site
+
+`science_configurations`
 
-    - [forty-two-site-test][forty-two-me]: to run simulations using 42 FLUXNET sites
-    - [five-site-test][five-me]: to run simulations using 5 FLUXNET sites
-    - AU-Tum: to run simulations at the Tumbarumba (AU) site
-    - AU-How: to run simulations at the Howard Spring (AU) site
-    - FI-Hyy: to run simulations at the Hyytiala (FI) site
-    - US-Var: to run simulations at the Vaira Ranch-Ione (US) site
-    - US-Whs: to run simulations at the Walnut Gulch Lucky Hills Shrub (US) site
+: User defined science configurations. This key is **optional** and can be omitted from the config file. If not specified, `science_configurations` is set internally to the following data structure:
```

Comment:
I would put the default science config on their own page and refer to them from here. Happy to leave this here and re-organise in a follow-up PR.

Reply:
It should state clearly these science configurations replace the default. They aren't additional configurations.

Reply:
See [ed73205](https://github.com/CABLE-LSM/benchcab/pull/51/commits/ed732050fd456f09512a9e56650d56d99f829c19).